### DISCUSSION TITLE
Connect second Xiaomi model ZNCZ12LM

### DISCUSSION THREAD

**Discussion by mrochette01** [Koenkk/zigbee2mqtt#13707] [Devices]
I installed 2 Xiaomi ZNCZ12LM (smart plug) routers. The router function of the first one works very well as shown in the attached graphic. For the second one, no Xiaomi Zigbee device connects to it. How can I get it detected so that the Zigbee network is more balanced.

There are 4 devices (the ones on the right) that connected to the controller and not to the Xiaomi ZNCZ12LM router. They are all Aqara (Xiaomi) devices like the others.

Thanks!
![2022-08-26_17-59-53](https://user-images.githubusercontent.com/90461724/186996453-e48b3499-b58b-46fd-84c8-ec98837fcfa4.jpg)

**Comment by jerrm** [comment]
With so few devices, I don't know if it's worth the effort if everything is currently working OK.

Aqara Temp/Humidity sensors will usually find a better route if they need to, but I don't know about the leak sensors.

Have you tried re-pairing the endpont(s) with the specific plug you want to use?  The "permit join" button is actually a drop-down and you can choose the specific router device you wish to put into pairing mode.

When re-pairing, you do not need to delete and re-add the device in z2m.  In z2m, just "permit join" for the router device you want to use, and then press the button on the Aqara endpoint until it goes back into paring mode.   That should force the endpoint to re-pair "in place" via the desired router, keeping name and settings you've already edited in z2m.

![image](https://user-images.githubusercontent.com/12073505/186999319-ac0cb468-27a1-442b-aa4d-e92b589f8623.png)

**Reply by mrochette01** [reply]
Hello, I have not tried this re-pairing option. I will make a test and confirm here the result.

**Reply by mrochette01** [reply]
Hello,
Here are the results of my tests.

I did the test. When I use #1 or #2, the device does not match. When I use the "All" option, then the device matches again with the "coordinator".

**Reply by mrochette01** [reply]
Also, I have validated the OTA of the 2 ZNCZ12LM. Their firmware is up to date.

**Reply by jerrm** [reply]
Not sure where to go, that should work.  I'd probably try pairing via the sonoff plugs just to see what happens.

**Reply by mrochette01** [reply]
I tried pairing with all 4 routers including sonoff and it didn't work.  The only option that works is ALL
